### ISSUE TITLE
CNN MNIST example probabilities only reported as 0 or 1

### ISSUE THREAD

**Issue by rwbarrette** [aymericdamien/TensorFlow-Examples#182]
So, I'm still new to tensor flow, and I really appreciate all of the tutorials and examples that you have produced.  My question has to do with the probabilities of the predictions during classification of a test set using your example.  When I add the following code to your example:

    `probs=tf.nn.softmax(logits)`
    `predProb` = sess.run(probs, feed_dict={X: mnist.test.images[:4],
                                      keep_prob: 1.0})
    `print(predProb)`

The output I get looks like:

[[ 0.  0.  0.  1.  0.  0.  0.  0.  0.  0.]
 [ 0.  0.  0.  1.  0.  0.  0.  0.  0.  0.]
 [ 0.  0.  0.  0.  0.  0.  0.  0.  0.  1.]
 [ 1.  0.  0.  0.  0.  0.  0.  0.  0.  0.]]

Isn't this what you would expect from argmax, while (I thought) softmax Should have probabilities that are <1 and >0 as opposed to a binary output?  If I am mistaken (which I am sure that I am), is there a way to obtain these probabilities in the context of your code?
